tests/drivers/interrupt_controller/intc_exti_stm32/dts/bindings/test-st-stm32-exti.yaml

```
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2025 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+    This binding provides resources required to build and run the
+    tests/drivers/interrupt_controller/intc_exti_stm32 test in Zephyr.
+
+compatible: "test-st-stm32-exti"
+
+properties:
+
+  exti-line-nr:
+    type: int
+    required: true
+    description: |
+      The EXTI line number to be used for the test.
```

Nit: slight reword
```suggestion
      Number of the EXTI line to use for the test.
```
\+ from my understanding, the following note should be added?
```suggestion
      Number of the EXTI line to use for the test.
      This line must support software-generated events/interrupts.
```
(I would need to cross-check with RM for exact wording)

---

For my understanding - all the EXTI lines supporting SW generated interrupts. Thus I prefer the first variant. Also this topic belongs to me more or less to documentation area. Of course you're free to reword it in the next PR's.